src/steps/writeReadme/generateTopContent.ts

```diff
@@ -1,83 +1,87 @@
 import { Options } from "../../shared/types.js";
 
 export function generateTopContent(options: Options, existingBadges: string[]) {
-	const remainingExistingBadges = new Set(existingBadges);
-	const badges: string[] = [];
+  const remainingExistingBadges = new Set(existingBadges);
```

Comment:
Same as https://github.com/JoshuaKGoldberg/create-typescript-app/pull/881#pullrequestreview-1640908590 - looks like formatting is being applied differently?

Reply:
Should be fixed